Jenkinsfile

```diff
@@ -4,25 +4,36 @@
 properties([buildDiscarder(logRotator(artifactNumToKeepStr: '20', numToKeepStr: '20'))])
 
 node {
-
+  stage 'Setup'
   deleteDir()
   checkout scm
   sh 'docker build -t blueocean_build_env --build-arg GID=$(id -g ${USER}) --build-arg UID=$(id -u ${USER}) - < Dockerfile.build'
 
   configFileProvider([configFile(fileId: 'blueocean-maven-settings', targetLocation: 'settings.xml')]) {
 
-  docker.image('blueocean_build_env').inside {
+  sh "./acceptance-tests/runner/scripts/start-selenium.sh"
```

Comment:
It starts a docker cotnainer which has to share the network with the main container